packages/wrangler/src/logger.ts

```diff
@@ -102,6 +102,7 @@ export class Logger {
 	}
 
 	private static _getBottomFloat?: () => string;
+	private static _previousBottomFloatLineCount = 0;
```

Comment:
Does this work as expected given logs are intermingled between Miniflare's logger and Wrangler's logger?